src/utils/utils.ts

```diff
@@ -425,20 +425,29 @@ function outputFormatterJson(json: string): string {
   return `${JSON.stringify(json, standardErrorReplacer)}\n`;
 }
 
-function outputFormatterError(err: Error): string {
+// Anything is throwable
+function outputFormatterError(err: any): string {
   let output = '';
   let indent = '  ';
   while (err != null) {
-    if (err instanceof networkErrors.ErrorPolykeyRemote) {
+    if (
+      err instanceof networkErrors.ErrorPolykeyRemote ||
+      err instanceof errors.ErrorPolykeyCLI
+    ) {
       output += `${err.name}: ${err.description}`;
       if (err.message && err.message !== '') {
         output += ` - ${err.message}`;
       }
-      if (err.metadata != null) {
+      if (
+        err instanceof networkErrors.ErrorPolykeyRemote &&
+        err.metadata != null
+      ) {
         output += '\n';
         for (const [key, value] of Object.entries(err.metadata)) {
           output += `${indent}${key}\t${value}\n`;
         }
+      } else {
+        output += '\n';
       }
       output += `${indent}timestamp\t${err.timestamp}\n`;
```

Comment:
Is the timestamp a part of the metadata? This will show for both `PolykeyCLI` errors and `PolykeyRemote` errors. I'm not sure if we need the entire exact timestamp of the error.